project.yaml

```diff
@@ -503,9 +503,10 @@ Layer:
       <<: *osm2pgsql
       table: |-
         (SELECT
-            way, landuse, leisure
+            way, COALESCE(landuse, CASE WHEN "natural" = 'wood' THEN 'forest' END) AS feature
```

Comment:
Yes, with the current WHERE clause it is equivalent to `COALESCE(landuse, 'forest')`